_haikus/2022_kubernetes_spider.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Kubernetes Spider
+author: Paskalouis
+---
+
+Like a spider web<br>
+Connecting functioning limb<br>
+Kubernetes, controls<br>
```

Comment:
This last line has one too many syllables

Reply:
Hi, I have revised the third line. Thanks!